Makefile

```diff
@@ -20,6 +20,8 @@ image-build:
 		--build-arg PKG=$(PKG) \
 		--build-arg SRC=$(SRC) \
 		--build-arg TAG=$(TAG) \
+		--build-arg MAJOR=$(shell ./scripts/version.sh ${TAG} major) \
+		--build-arg MINOR=$(shell ./scripts/version.sh ${TAG} minor) \
```

Comment:
This `scripts/version.sh` doesn't leverage assumptions/implications established in the build process of all of our other projects (we source it and process results in other scripts). As such, doing something different as is spelled out here is fine but we should call it something different so as to avoid confusion with established patterns. I suggest `scripts/version-parse` or `scripts/semver-parse`.